### PR TITLE
affichage.html

### PR DESCRIPTION
**Pull request by Tomo59**

### CODE REVIEW

File: couchapps/bib_input/_attachments/affichage.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  <body>
+    <div id="bib_output_title"></div>
+    <div style="clear:both"></div>
+    <div id="bib_output_last_contestant_infos"></div>
+    <div id="bib_output_last_contestants" class="bottom_pane"></div>
+
+  </body>
+  <script src="../common/vendor/couchapp/loader.js"></script>
+  <script type="text/javascript" charset="utf-8">
+    $(document).ready(function(){
```
Comment (jonenst) [inline couchapps/bib_input/_attachments/affichage.html:17]: no arrow here ?

File: couchapps/bib_input/evently/bib_output_last_contestants/_changes/opts.json [hunk 2]
```diff
@@ -0,0 +1,4 @@
+{
+  "id":"checkpoints",
+  "filter": "bib_input/checkpoints"
```
Comment (jonenst) [inline couchapps/bib_input/evently/bib_output_last_contestants/_changes/opts.json:3]: needs the corresponding
$.evently.changesDBs[app.db.name]["checkpoints"].opts.site_id = app.site_id;

File: couchapps/bib_input/_attachments/affichage.html (hunk 1 above)
Comment (Tomo59) [reply couchapps/bib_input/_attachments/affichage.html]: fixed

File: couchapps/bib_input/evently/bib_output_last_contestants/_changes/opts.json (hunk 2 above)
Comment (Tomo59) [reply couchapps/bib_input/evently/bib_output_last_contestants/_changes/opts.json:3]: fixed

File: couchapps/bib_input/_attachments/affichage.html [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+    <div style="clear:both"></div>
+    <div id="bib_output_last_contestant_infos"></div>
+    <div id="bib_output_last_contestants" class="bottom_pane"></div>
+
+  </body>
+  <script src="../common/vendor/couchapp/loader.js"></script>
+  <script type="text/javascript" charset="utf-8">
+    $.couch.app(function(app) {
+      $.evently.changesOpts.filter = "bib_input/no-ping";
+      app.bibs_clocked_here = [];
```
Comment (jonenst) [inline couchapps/bib_input/_attachments/affichage.html:19]: to remove

**Review by jonenst — APPROVED** [review]

**Pull request merged**